### PR TITLE
mm-461 Trailing space in my access_token causes uninformative error message fixes #461

### LINKED ISSUE #461: Trailing space in my access_token causes uninformative error message
Hi, I just Installed this plugin `oauth2: server response missing access_token`. How to resolve? do I need to renew? I've created this token just now.

-----

Turns out, I had trailing space in my access_token. It was really bad error message though, should be fixed.

_Originally posted by @sanjaydemansol in https://github.com/mattermost/mattermost-plugin-github/issues/431#issuecomment-883065815_

### PR DESCRIPTION
**Pull request by sibasankarnayak**
Removed trailing spaces from clientid , accesstoken and orgname while geting the configuration

ticket here
https://github.com/mattermost/mattermost-plugin-github/issues/461

### CODE REVIEW

**Comment by sibasankarnayak** [comment]
> Hello @sibasankarnayak,
> 
> Thanks for your pull request! A Core Committer will review your pull request soon. For code contributions, you can learn more about the review process [here](https://developers.mattermost.com/contribute/getting-started/code-review/).
> 
> Per the [Mattermost Contribution Guide](https://developers.mattermost.com/contribute/getting-started/), we need to add you to the list of [approved contributors](https://docs.google.com/spreadsheets/d/1NTCeG-iL_VS9bFqtmHSfwETo5f-8MQ7oMDE5IUYJi_Y/pubhtml?gid=0&single=true) for the Mattermost project.
> 
> **Please help complete the Mattermost [contribution license agreement](http://www.mattermost.org/mattermost-contributor-agreement/)?**
> Once you have signed the CLA, please comment with `/check-cla` and confirm that the CLA check is green.
> 
> This is a standard procedure for many open source projects.
> 
> Please let us know if you have any questions.
> 
> We are very happy to have you join our growing community! If you're not yet a member, please consider joining our [Contributors community channel](https://pre-release.mattermost.com/core/channels/tickets) to meet other contributors and discuss new opportunities with the core team.

/check-cla

File: server/client/client.go [hunk 1]
```diff
@@ -83,13 +84,18 @@ func (c *Client) GetConfiguration() (*plugin.Configuration, error) {
 
 	config := &plugin.Configuration{}
 	err = json.Unmarshal(respBody, config)
+	config.GitHubOrg = TrimSpace(config.GitHubOrg)
```
Comment (hanzei) [inline server/client/client.go:87]: Can these just be `strings.TrimSpace`?

**Review by hanzei — APPROVED** [review]
LGTM :+1: Thanks for the improvment

File: server/client/client.go (hunk 1 above)
Comment (sibasankarnayak) [reply server/client/client.go:87]: Sure

**Review by mickmister — APPROVED** [review]
LGTM

**Pull request merged**